Documentation/DataIngress/OMF_Ingress_Topics.md

```diff
@@ -217,23 +198,22 @@ Updates an existing topic. Only the topic name and description can be updated.
 ``namespaceId``
   Unique Id for the namespace. 
 ``topicId``
-  Topic Id for the topic to be updated.
+  Topic Id for the Topic to be updated.
```

Comment:
Not exactly sure how to do this, but can you try to find a way to document that you need the object ID in the body and the route for a PUT call to a topic or subscription, but it is not allowed in the POST call?

Reply:
Fixed.